CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # CHANGELOG
 
+# Version 3.2.0-BETA.2
+* There were lots of refactorings.
+* Enable profiler by default.
```

Comment:
the profiler is still proof-of-concept (and undocumented), so let's not mention it (also, it's not enabled by default, it's only included by default)
```suggestion
```